common/etc/nginx/include/s3gateway.js

```diff
@@ -643,10 +683,108 @@ function _require_env_var(envVarName) {
     }
 }
 
+var maxValidityOffset = 270000;
```

Comment:
It would be good to add a comment or change the variable name here to indicate the unit of measurement for the time offset interval.

Reply:
👍